tests/test_main.py

```diff
@@ -1,6 +1,136 @@
-# TODO: add tests
-from app.main import hello_world
+import pytest
 
+from app.main import make_a_person
+from app.main import Person
 
-def test_hello_world():
-    assert hello_world() == "Hello, world!"
+
+def test_person_class_attribute_people_exists():
+    assert hasattr(Person, 'people'), (
+        "Class Person should have class attribute 'people'"
+    )
+    assert len(Person.people) == 0, (
+        "Initial length of 'Person.people' should equal to 0"
+    )
+
+
+@pytest.mark.parametrize(
+    'person,name,age',
+    [
+        ({'name': 'Ross', 'age': 30, 'wife': 'Rachel'}, 'Ross', 30),
```

Comment:
Strange test, why you pass `dict` to tests as `person`, if you don't need it - pass just `name` and `age`, if you just checking constructor